example/App.tsx

```
@@ -1,76 +1,343 @@
+      ExpoInAppUpdates.addUpdateListener("updateCancelled", (event) => {
+        appendEvent(event.reason ?? "Update cancelled");
+      }),
+      ExpoInAppUpdates.addUpdateListener("updateCompleted", () => {
+        appendEvent("Update completed");
+      }),
+    ];
 
-    // Start the update check process
-    checkUpdates();
+    return () => {
+      removeListeners.forEach((remove) => remove());
```

Biome (2.4.16)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)